database/seeders/DatabaseSeeder.php

```diff
@@ -63,5 +64,11 @@ public function run()
 
         MenuItem::truncate();
         MenuItem::factory()->count(7)->create();
+
+        MenuItem::truncate();
+        MenuItem::factory()->count(7)->create();
```

Comment:
I see we create 7 MenuItems, twice. Was this intended or duplication?

Reply:
It was mistake, fixed 👌